linera-chain/src/unit_tests/chain_tests.rs

```diff
@@ -111,7 +111,7 @@ async fn test_block_size_limit() {
     let mut chain = ChainStateView::new(chain_id).await;
 
     // The size of the executed valid block below.
-    let maximum_executed_block_size = 691;
+    let maximum_executed_block_size = 699;
```

Comment:
Why did this got increased?

Reply:
The `OpenChain` message below contains a `Committee`, which contains a `Policy`, which now has an additional field that is 8 bytes wide.